js/interface.js

```diff
@@ -337,15 +337,9 @@ function fetchCurrentDataSourceEntries(entries) {
     } else {
       $('#show-versions').show();
 
-      // Add an empty object in the beginning for _.extend
-      rows.unshift({});
-
-      var computedColumns = _.keys(_.extend.apply({}, rows.map(function(row) {
-        return row.data;
-      })));
-
-      // Remove the first entry
-      rows.shift();
+      let flatObj1 = {};
```

Comment:
Can we use self-descriptive name for this variable?  Also, use the uniform variable declaration keywords. E.g if using `var` every ever, please use `var` only.